### PR TITLE
estimateDeposit api for transparency + ux

### PR DESCRIPTION
**Pull request by carsonfarmer**
No rush on this one, but provides a useful API extension that makes estimating deposit sizes much easier for humans 🤖.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,9 +73,9 @@ await wallet.signOut();
 
 ```typescript
 import { providers } from "ethers";
-import { init } from "@textile/eth-storage";
+import { init, requestSignIn } from "@textile/eth-storage";
 
-await window.ethereum.enable();
+await requestSignIn();
```
Comment (andrewxhill) [inline README.md:78]: though i like the abstraction, what it gains in simplicity i wonder if we are losing in interop/familiarity. as in, interacting with the ethereum object is a standard process in documentation for any dapp building. here, we have to explain something new/different than every other tutorial they might read.

**Review by andrewxhill — APPROVED** [review]
lgtm. just one comment about a concern we are being too helpful and not just matching what the rest of the libs do.

File: README.md (hunk 1 above)
Comment (carsonfarmer) [reply README.md:78]: actually, I totally agree with you! thanks for flagging that. in hindsight, I think you're right. I'm going to leave the api available, but I'm going to revert the docs to use the more "standard" approach

Comment (andrewxhill) [reply README.md]: sweet

**Pull request merged**